documentation/milestones/individual_milestones/search_mvp.md

```diff
@@ -0,0 +1,218 @@
+
+- How should search and filter criteria be passed to the API endpoint used for search?
+- How should we support basic logic such as greater than, less than, as well as and/or/not conditions?
+- Which fields should support filtering and sorting?
+- Do we use an existing search service (e.g. Elasticsearch) or build our own?
+
+### Search Page
+
+The primary page we'll need to deliver for the user interface (UI) is a search page, which should allow users to:
+
+- Enter key words to search the list of funding opportunities
+- Filter search results by *at least* one structured field (e.g. award date, opportunity type, etc.)
+- View a common set of fields for each opportunity returned by the search (e.g. title, award date, summary, etc.)
+- Sort the set of results returned by *at least* one structured field (e.g. award date, title, etc.)
+
+### Opportunity Listing Page
+
+The secondary page we want to deliver within the user interface (UI) is an opportunity listing page, which should:
+
+- Display a consistent list of metadata about each opportunity
+- Use only fields that are available in the existing grants.gov cloud database
+- Link to the corresponding opportunity in the existing grant.gov interface
+
+**Note:** This is a stretch goal for this deliverable and may be substituted with a direct link to the grants.gov page for a given opportunity from the search results.
+
+### Definition of done
+<!-- Required -->
+
+- [ ] Basic requirements:
+  - [ ] Code is deployed to `main` & PROD through our CI/CD pipeline
+  - [ ] Services are live in PROD (may be behind feature flag)
+  - [ ] Translations are live in PROD (if necessary)
+  - [ ] All new services have passed a security review (if necessary)
+  - [ ] All new services have completed a 508 compliance review (if necessary)
+  - [ ] Data needed for metrics is actively being captured in PROD
+  - [ ] Key architectural decisions made about this deliverable are documented publicly
+- [ ] Functional requirements:
+  - [ ] Users can search for opportunities by key word
+  - [ ] Users can filter search results by *at least one* structured field (e.g. award date, opportunity type, etc.)
+  - [ ] Users can sort search results by *at least* one structured field (e.g. award date, title, etc.)
+  - [ ] All of the search features available in the UI are also available via the API, and vice versa
+  - [ ] Search criteria are reflected in the URL so that users can bookmark or share the link to a given set of search results
+  - [ ] Users can access the corresponding grants.gov page for an opportunity they find on simpler.grants.gov via link on the search results or opportunity listing page
+
+### Proposed metrics for measuring goals/value/definition of done
+<!-- Required -->
+
+1. Total number of searches made via the UI
+2. Total number of searches made via the API
+3. Number of results returned per search
+4. Frequency distribution of keyword search terms
```

Comment:
@widal001 What do you mean by this metric?

Reply:
We'll want to know what the _most_ and _least_ common search terms are, e.g. are a lot of people searching for "environmental subsidy" or "infrastructure". Having a frequency distribution of searches by keyword would help us map trends in the content and format of searches

Feel free to reword this in a way that makes more sense to y'all